skore/tests/unit/item/test_pandas_dataframe_item.py

```diff
@@ -53,7 +53,7 @@ def test_dataframe_with_complex_object(self, mock_nowstr):
         dataframe = DataFrame([{"key": np.array([1])}], Index([0], name="myIndex"))
         item = PandasDataFrameItem.factory(dataframe)
 
-        assert type(item.dataframe["key"].iloc[0]) is list
+        assert isinstance(item.dataframe["key"].iloc[0], list)
```

Comment:
Please revert this changes: `np.array` is an instance of list.

Reply:
uh, oh, sorry... ^^'